content/docs/user-guide/testing/getting-started/_index.md

```diff
@@ -0,0 +1,233 @@
+  - Helps monitor for application crashes.
+  - Test file names should start with `test_`.
+  - Uses the same Python interpreter that launches when you run `o3de/python/python.sh` or `o3de\python\python.cmd`.
+
+- Test embedded inside O3DE Editor
+  - Use in `EditorPythonBindings` tests which target workflows inside O3DE Editor.
+  - Helps expose editor functionality to a test script.
+  - Test file names should ***not*** include affixes such as `test_` or `_tests`.
+  - PyTest is not used inside the editor, and most PyTest functionality is not available
+    - PyTest is still initiated by the external Python interpreter instance, to handle Editor crashes.
+
+### Registering a new Python test
+
+Registering a Python-based test is simpler than registering a C++ test. However it still requires defining a C++ library that will be tested, which is often already complete before designing integration tests. The steps below assume you have already defined a production library. You can read more about defining production code in [Getting Started with the CMake Build System](/docs/user-guide/build/).
+
+#### Step 1: Register a PyTest target
+
+Find the `CMakeLists.txt` that defines the system you are testing. It should exist at a path similar to `o3de/.../<MyModule>/CMakeLists.txt`. Note that it may be appropriate to register broader tests in a parent directory's CMakeLists.txt instead. If you are creating a test that is specific to a game project, you should register it in a `CMakeLists.txt` in that game project. When you find the right place for the test to be registered, add a line similar to:
+
+```
+ly_add_pytest(
+    NAME myPythonTest 
+    PATH ${CMAKE_CURRENT_LIST_DIR}/Tests/test_MyFile.py
+    TEST_SERIAL # many larger-than-unit tests can interfere with one another in parallel
+    RUNTIME_DEPENDENCIES # helps the test build-and-run from an IDE
+        Legacy::Editor
+        AZ::AssetProcessor
+        AutomatedTesting.Assets
+)
+```
+
+To verify everything is set up correctly, run the [CMake configure command](/docs/user-guide/build/configure-and-build/) from the CMake CLI or CMake GUI. This will register everything you just added, and emit errors if anything was misconfigured.
+
+#### Step 2: Write new Python tests
+
+These tests primarily use standard [Python](https://docs.python.org/3/) code along with [PyTest](https://docs.pytest.org/). Depending on where your test is focused, it should likely include one of two automation libraries:
+
+Internal tests of O3DE Editor functionality should use [EditorPythonBindings](/docs/user-guide/testing/parallel-pattern/). This helps expose editor C++ interfaces to Python. These tests almost always require the `TEST_SERIAL` flag.
+
+External tests at the Operating System level often use the [LyTestTools](/docs/user-guide/testing/lytesttools/) module, which simplifies interacting with O3DE applications. These tests often use the `TEST_SERIAL` flag.
```

Comment:
- Consider putting into bullets and restructuring the sentences to name the automation libraries first. This is because the previous sentence introducing them says "it may include one of two automation libraries", so I was expecting to see the names of the libraries first. 
- I vaguely understand the difference between "almost always require" and "often use" regarding the `TEST_SERIAL` flag. I definitely think more information is needed to understand the context here, like: What does the `TEST_SERIAL` flag do? When do internal tests use them? When do external tests use them? You don't have to answer that here, maybe you go into it in another doc. But still I don't think it gives the reader much without the additional details. Maybe all they need to know in this doc is that "both internal and external tests may use the `TEST_SERIAL` flag. Specifying how often but leaving out a full explanation can cause confusion. 
```suggestion
- [EditorPythonBindings](/docs/user-guide/testing/parallel-pattern/) for internal tests on the O3DE Editor functionality. 

- [LyTestTools](/docs/user-guide/testing/lytesttools/) module for external tests at the Operating System level.

Both internal and external tests may use the `TEST_SERIAL` flag. 
```